forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -291,7 +291,7 @@ public Optional<String> fileContents(String filename, String ref) {
     }
 
     @Override
-    public void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail) {
+    public void writeFileContents(String filename, String content, Branch branch, String message, String authorName, String authorEmail, boolean createNewEmail) {
```

Comment:
This variable name doesn't make sense here.

Reply:
Thanks for catching it! I was in a hurry when working on it